Makefile

```diff
@@ -0,0 +1,66 @@
+artifact_name       := company-documents.web.ch.gov.uk
+
+.PHONY: all
+all: build
+
+.PHONY: clean
+clean:
+	mvn clean
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+
+.PHONY: build
+build:
+	mvn install
```

Comment:
Should this be mvn compile ? https://github.com/companieshouse/styleguides/blob/master/Makefiles/java_makefiles.md